web/config/staging/core.entity_form_display.node.global_training_day_registration.default.yml

```diff
@@ -0,0 +1,144 @@
+uuid: 7c2a20ba-0d5f-4577-8639-654d1a39aebe
```

Comment:
I think the composer json and lock file with requiring the country module is missing.